src/oph/ehoks/virkailija/handler.clj

```diff
@@ -203,13 +203,13 @@
         :audit-data {:new hoks}))))
 
 (defn opiskeluoikeus-void-or-active?
-  "Check whether opiskeluoikeus is nil or active"
-  [opiskeluoikeus-oid]
-  (let [opiskeluoikeus (koski/get-opiskeluoikeus-info opiskeluoikeus-oid)]
-    (or
-      (nil? opiskeluoikeus)
-      (not (op/opiskeluoikeus-tila-inactive?
-             (op/get-opiskeluoikeus-tila opiskeluoikeus))))))
+  "Check whether opiskeluoikeus is nil (mitätöity or otherwise not found) or
+   active"
+  [opiskeluoikeus]
+  (or
+    (nil? opiskeluoikeus)
+    (not (op/opiskeluoikeus-tila-inactive?
```

Comment:
How does this work? With quick glance it seems the function should return `false` when `opiskeluoikeus` is active and `true` otherwise, i.e. when `opiskeluoikeus is not found (is `nil`) or when `opiskeluoikeus-tila` is inactive? To me it seems the `not` is not correct here, but I could be mistaking. :thinking: 

Also, there seems to be a function called `opiskeluoikeus-active?` in `oppijaindex.clj` that has somewhat similar functionality, with the difference that it takes `opiskeluoikeus-oid` as a parameter and returns `true` when `opiskeluoikeus` is active. I would keep either function and use naming like `opiskeluoikeus-active?` because that explicitly hints the function returns `true` when `opiskeluoikeus` is found and is active.

Reply:
Good catch! I did some refactoring to get rid of the "duplicate" function